qa/Makefile.am

```diff
@@ -1,2 +1,2 @@
 SUBDIRS = coccinelle
-EXTRA_DIST = wirefuzz.pl sock_to_gzip_file.py drmemory.suppress
```

Comment:
Why is this file being deleted?

Reply:
1. Final goal is to replace pcre by pcre2
2. drmemory.suppress references pcre
3. Victor told me this was no longer used and is obsolete